jordan_bundy/test/greet_test.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var expect = require('chai').expect;
+var greet = require(__dirname + '/../lib/greet');
+// Create a dummy process to test the command line capability
+var process = {argv: ['node', 'greet.js', 'some name']};
```

Comment:
Another issue that I went round and round on!  The problem I saw was that if I pass process.arg[2] into my test, then my test code is the one processing the argument instead of my function.  Again, the bonus question was vague, but I thought the goal was to test wether or not our function was doing the work.